spotless/eclipse-java-formatter.xml

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="16">
+    <profile kind="CodeFormatterProfile" name="Spotless" version="16">
```

Comment:
This probably doesn't fall under "formatting" but is there a way to add `@Override` for all overriding methods? I.e., when the compiler sees a method that is overriding  a parent method but is not marked with `@Override`, I'd like to auto-add `@Override` on the previous line.